docs/articles/instances.md

```diff
@@ -0,0 +1,26 @@
+# Instances
+
+## What is an Instance?
+Instances are the Marathon representation of [⇒Tasks](tasks.md) in Mesos. An Instance can exist even if there is currently no Task associated. Depending on the goal of a given Instance, Marathon will take actions to reach that goal: it might match offers in order to launch a Task or [⇒TaskGroup](taskGroups.md), or it might send kill requests to Mesos and eventually expunge the Instance from its state. See [⇒Instance Life Cycle](instanceLifeCycle.md) for more information.
```

Comment:
I think we should spend a few additional words on why this layer is meaningful e.g. something along the lines of:
```There are a few reasons why this additional layer of indirection between actual Mesos tasks and users is meaningful. First and foremost: an Instance can have one task (see [Apps](apps.md)), two or more tasks (see [Pods](pods.md)) or no tasks at all (if the task was killed). 
The connection between Intances and it's tasks is quickly visible in their names. An app with name `/my-service` will have an instance named `my-service.<UUID>` and a task `my-service.<UUID>.3` where `.3` means that this is the third incarnation of this instance (previous two were killed or failed for some reason)```